.babelrc

```diff
@@ -1,10 +1,5 @@
 {
   "presets": [
-    ["latest", {
```

Comment:
Any reason for removing this? webpack 2 relies on modules: false so that webpack itself can transform import statements, and System.import statements, etc.